integration_test/lib/vnspec/vm.rb

```diff
@@ -248,6 +256,10 @@ def ipv4_address
         end
       end
 
+      def change_ipv4_address(new_address, new_prefix = 24)
```

Comment:
If you need this method it's ok. But it should be included in another PR I think.